lib/tina/restore_plan.rb

```diff
@@ -46,13 +46,15 @@ def size
 
       def chunk(max_chunk_size)
         @chunks ||= begin
```

Comment:
Memoizing the result without respecting argument is a accident waiting to happen